### ISSUE TITLE
how to fix a mistake

### ISSUE THREAD

**Issue by Hovik99** [oklai/koala#795]
**Koala version**

**Bug report or feature request?**

how to fix a mistake
![default](https://user-images.githubusercontent.com/38872218/53666976-0e52bc00-3c89-11e9-9ed1-1a2ccbe71f7a.JPG)
![2](https://user-images.githubusercontent.com/38872218/53666977-0e52bc00-3c89-11e9-8ea3-691814887f58.JPG)

**Comment by 5572189** [comment]
How to solve

**Comment by zaygraveyard** [comment]
Duplicate of #740?
You can try deleting `%USERPROFILE%/.koala/settings.json`

**Comment by 5572189** [comment]
> 重复[＃740](https://github.com/oklai/koala/issues/740)？
> 您可以尝试删除`%USERPROFILE%/.koala/settings.json`

I deleted this file, but it still can't work. This file will be generated automatically

**Comment by zaygraveyard** [comment]
@5572189 Right after deleting this file, does Koala start? If not what is the error?

**Comment by 5572189** [comment]
> @ 5572189删除此文件后，考拉开始了吗？如果不是什么错误？



> @5572189 Right after deleting this file, does Koala start? If not what is the error?

May I ask what causes this problem

**Comment by zaygraveyard** [comment]
@5572189 I don't know
I'm trying to understand the problem that you're having
And I need your help, so please answer the questions above

**Comment by 5572189** [comment]
> @5572189 I don't know
> I'm trying to understand the problem that you're having
> And I need your help, so please answer the questions above

你会说中文吗？上面的问题就是我把这个setting.json删除之后，把项目添加上去，这个setting.json会自动生成，如果先把项目添加上去，再去删除这个setting.json,koala依然抱着个错误，但是可以执行成功

**Comment by zaygraveyard** [comment]
@5572189 Sorry no I don't speak Chinese
Is it the same error as in the screenshots above?

**Comment by 5572189** [comment]
> @5572189 Sorry no I don't speak Chinese
> Is it the same error as in the screenshots above?

Yes, and the above chart, thanks for the patience to explain, I've removed all about the koala, the new downloaded from the official website, installed with no problems, but the Green Box compilation succeeds, not out.Again Thank you for the patience to explain, thanks

**Comment by 5572189** [comment]
> @5572189 Sorry no I don't speak Chinese
> Is it the same error as in the screenshots above?

So, I want to know what causes the problem

**Comment by zaygraveyard** [comment]
@5572189 Unfortunately I don't know the cause :sweat: 
Can you post here the contents of your `%USERPROFILE%/.koala/settings.json` that causes this error?

I can try to fix it regardless, it would be much better to understand the root cause

**Comment by 5572189** [comment]
> @5572189 Unfortunately I don't know the cause 😓
> Can you post here the contents of your `%USERPROFILE%/.koala/settings.json` that causes this error?
> 
> I can try to fix it regardless, it would be much better to understand the root cause

It doesn't matter, new installation, also can, just did not compile successful clew
